content/comm/conferences/index.en.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "R-Ladies Conference Participation"
+menuTitle: "Conference Participation"
+weight: 110
+---
+
+R-Ladies participates in conference sessions such as contributed talks,
+panel discussions, networking events, poster presentations, and booths to 
+increase awareness of our organization.
+
+## Organizing a conference event
+
+If you have an R-Ladies member interested in leading a conference
```

Comment:
```suggestion
If you are an R-Ladies member interested in leading a conference
```

So one only needs to be a member but what does this mean exactly, e.g. "listed in the directory", or organizer/global team member?